File: app/src/main/java/com/tokeninc/sardis/application_template/helpers/PrintHelpers/DateUtil.java
package com.tokeninc.sardis.application_template.helpers.PrintHelpers;

import java.text.SimpleDateFormat;
import java.util.Calendar;
import java.util.Date;
import java.util.Locale;

public class DateUtil {

    public static String getDate(String format) {
        Date calDate = Calendar.getInstance().getTime();
        return new SimpleDateFormat(format, Locale.getDefault()).format(calDate);
    }

    public static String getTime(String format) {
        Date calDate = Calendar.getInstance().getTime();
        return new SimpleDateFormat(format, Locale.getDefault()).format(calDate);
    }
}

File: app/src/main/java/com/tokeninc/sardis/application_template/helpers/printHelpers/DateUtil.kt
package com.tokeninc.sardis.application_template.helpers.printHelpers

import java.text.SimpleDateFormat
import java.util.*

class DateUtil {

    fun getDate(format: String?): String? {
        val calDate: Date = Calendar.getInstance().time
        return SimpleDateFormat(format, Locale.getDefault()).format(calDate)
    }

    fun getTime(format: String?): String? {
        val calDate: Date = Calendar.getInstance().time
        return SimpleDateFormat(format, Locale.getDefault()).format(calDate)
    }

}